src/lib/sockets.ts

```diff
@@ -161,143 +161,3 @@ export default {
 };
 
 (window as any)["socket"] = socket;
-
```

Comment:
I'm not sure what you mean here.   devel still has the experimental socket stuff in it, but anoek said he was going to take it out.  So I did :)  (because it breaks the tests)

Reply:
Ohh I thought he was just going to remove PerformanceObserver code, not all the test_socket stuff.

> I'm not sure what you mean here.

My bad, I assumed thus PR was behind https://github.com/online-go/online-go.com/commit/8428b9d7591e432d1266f179520423ae0035f299, now I understand this was removed on purpose

Reply:
Mmm - it should be in a separate PR of course.

I _think_ that I took out "only what was both breaking jest and targetting device performance".